### ISSUE TITLE
[FEATURE] better styling for Quote reply

### ISSUE THREAD

**Issue by Dun-sin** [Dun-sin/Whisper#346]
### Description

The styling looks bad, the quoted reply should take the whole width, have similar padding to the regular message, they also shouldn't be close to each other maybe a gap of 2(gap-2).

### Screenshots

![image](https://github.com/Dun-sin/Whisper/assets/78784850/1e125047-2b5e-425e-9919-e5e456c081f2)

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

**Comment by AKASH-PRASAD7** [comment]
Hey, I would like to work on this issue, would be great if you assign me this issue.

**Comment by Dun-sin** [comment]
> Hey, I would like to work on this issue, would be great if you assign me this issue.

Definitely, thanks for wanting to contribute, Make sure you follow the rules [👉🏽here👈🏽](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md#rules), else your PR will not be accepted and will be closed. Good Luck

**Comment by AKASH-PRASAD7** [comment]
> @AKASH-PRASAD7 it's been 7 days and there's no update for this

I'm sorry, but it seems like there's no one available to chat on the local server at the moment. Unfortunately, this means that I'm unable to produce the issue you're experiencing locally. Is there anything else I can assist you with?

**Comment by AKASH-PRASAD7** [comment]
> @AKASH-PRASAD7 it's been 7 days and there's no update for this

 I'm unable to produce the issue you're experiencing locally. Because I can't chat with two accounts on the same server locally. Is there anything else I can assist you with?

**Comment by AKASH-PRASAD7** [comment]
> > > @AKASH-PRASAD7 it's been 7 days and there's no update for this
> > 
> > 
> > I'm unable to produce the issue you're experiencing. Because I can't chat with two accounts on the same server locally. Is there anything else I can assist you with?
> 
> please read the readme, everything, and should have let me know a long time ago

Yeah I know, I've already read the README and successfully set up the project locally. However, I'm unsure how to initiate a conversation with someone else to reproduce the issue or obtain a quoted reply that seems to be unachievable locally.

**Comment by Dun-sin** [comment]
@AKASH-PRASAD7 been inactive for too long

**Comment by nayakswadhin** [comment]
Can you please assign this issue to me?? I would love to work on this issue :)

**Comment by Kritika30032002** [comment]
Plzz assign me this

**Comment by Dun-sin** [comment]
@Kritika30032002 according to the contributing guideline, your time is up and if there's a PR by the end of the day, you will be unassigned
